clpy/backend/function.pyx

```diff
@@ -34,6 +34,9 @@ cdef struct _CArray:
 cdef struct _CArray0:
     char unused
 
+cdef set _pointer_numpy_types = {numpy.dtype(i).type
+                                 for i in '?bhilqBHILQefdFD'}
```

Comment:
Why don't you use [the defined types](https://github.com/fixstars/clpy/blob/ccc1a3faaa040f814e436e8cfe361a7c52044c70/clpy/core/elementwise.pxi#L67)?

Reply:
I see. To use ```_python_scalar_type_set``` from a different file, I think the definition should be moved from ```elementwise.pxi``` to a more "generic" place.
I'm looking for a fitting place, and if someone could give me an instant idea I would appreciate it.

Reply:
Put the declaration of the Numpy scalar set in ```core.pxd```.